content/2021/2020-01-21_whats-new/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "What has 2020 brought to us?"
```

Comment:
```suggestion
title = "Updates from the past year"
```

"2020 brought" reminds me of COVID.

Can you change the file name as well? And the date in the filename needs to be fixed to use 2021.

Reply:
The date in the file name was a typo.